main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -542,8 +555,8 @@ object Weeder extends Phase[ParsedAst.Program, WeededAst.Program] {
       //
       // let* x = exp1; exp2     ==>   flatMap(x -> exp2)(exp1)
       //
-      val qname = Name.mkQName(Name.Ident(SourcePosition.Unknown, "flatMap", SourcePosition.Unknown))
-      val flatMap = WeededAst.Expression.VarOrDefOrSig(qname, loc)
+      val ident = Name.Ident(SourcePosition.Unknown, "flatMap", SourcePosition.Unknown)
+      val flatMap = WeededAst.Expression.VarOrDefOrSig(ident, loc)
```

Comment:
So this means that flatMap can from a local var or a use of a def or typeclass. Cool!